harper: 0.71.0 -> 1.0.0

ddogfoodd opened the pull request:
Changelog: https://github.com/Automattic/harper/releases/tag/v1.0.0
Diff: https://github.com/Automattic/harper/compare/v0.71.0...v1.0.0

## Things done
- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

ddogfoodd committed:
harper: 0.71.0 -> 0.72.0

Changelog: https://github.com/Automattic/harper/releases/tag/v0.72.0
Diff: https://github.com/Automattic/harper/compare/v0.71.0...v0.72.0

sumnerevans reviewed (approved):
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 461668`
Commit: `ee3da9bd05aa23219bfe9bea46bb63e4e17a7f36`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>harper</li>
    <li>vscode-extensions.elijah-potter.harper</li>
  </ul>
</details>

ddogfoodd committed:
harper: 0.72.0 -> 0.73.0

Changelog: https://github.com/Automattic/harper/releases/tag/v0.73.0
Diff: https://github.com/Automattic/harper/compare/v0.72.0...v0.73.0

ddogfoodd committed:
harper: 0.73.0 -> 1.0.0

Changelog: https://github.com/Automattic/harper/releases/tag/v1.0.0
Diff: https://github.com/Automattic/harper/compare/v0.73.0...v1.0.0

Aleksanaa commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 461668`
Commit: [`308c0cbf1c5125249e19290d3b850842ea8d2836`](https://github.com/NixOS/nixpkgs/commit/308c0cbf1c5125249e19290d3b850842ea8d2836) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/308c0cbf1c5125249e19290d3b850842ea8d2836..pull/461668/head))
Merge: [`819481315381ef6d5a220f46880bd7ac0d769d38`](https://github.com/NixOS/nixpkgs/commit/819481315381ef6d5a220f46880bd7ac0d769d38)

Logs: https://github.com/Aleksanaa/nixpkgs-review-gha/actions/runs/19809420135


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>harper</li>
    <li>vscode-extensions.elijah-potter.harper</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>vscode-extensions.elijah-potter.harper</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>harper</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>vscode-extensions.elijah-potter.harper</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>harper</li>
  </ul>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>harper</li>
    <li>vscode-extensions.elijah-potter.harper</li>
  </ul>
</details>

MasterEvarior reviewed (approved):
Approved automatically following the successful run of `nixpkgs-review`.
